### ISSUE TITLE
Feature Request: Make Payments block compatible with Dolphin Supernova screen reader

### ISSUE THREAD

**Issue by lakellie** [Automattic/jetpack#21458]
### Impacted plugin

Jetpack

### What

We received a report from a user in 4386637-zen letting us know the Payments block does not work with the Dolphin Supernova screen reader (which is one of the most popular screen readers).

### How

We would need to test and confirm how/if the block works with this and other screen readers. I am not sure what sort of accessibility testing is done when the blocks are originally developed.

**Comment by lakellie** [comment]
I am closing this. There was a miscommunication in the ticket and the issue is not related to the payments block.
